native/explorer/src/dataframe.rs

```diff
@@ -378,17 +390,28 @@ pub fn df_drop_duplicates(
 }
 
 #[rustler::nif(schedule = "DirtyCpu")]
-pub fn df_to_dummies(data: ExDataFrame) -> Result<ExDataFrame, ExplorerError> {
+pub fn df_to_dummies(
+    data: ExDataFrame,
+    selection: Vec<&str>,
+) -> Result<ExDataFrame, ExplorerError> {
     let df = &data.resource.0;
-    let new_df = df.to_dummies()?;
+    let new_df = df.select(&selection).and_then(|df| df.to_dummies())?;
     Ok(ExDataFrame::new(new_df))
 }
 
 #[rustler::nif(schedule = "DirtyCpu")]
-pub fn df_with_column(data: ExDataFrame, column: ExSeries) -> Result<ExDataFrame, ExplorerError> {
+pub fn df_with_columns(
+    data: ExDataFrame,
+    columns: Vec<ExSeries>,
+) -> Result<ExDataFrame, ExplorerError> {
```

Comment:
I think this one might be more efficient with an hstack.

Reply:
Just realised that won't work with a mutate on the same name as an old column, so this implementation is all good. :)